deploy-agent/deployd/common/config.py

```diff
@@ -105,7 +105,7 @@ def update_variables(self, deploy_status):
         if deploy_status.report.stageName:
             self._environ['STAGE_NAME'] = deploy_status.report.stageName
         if deploy_status.report.stageType:
-            self._environ['COMPUTE_STAGE_TYPE'] = deploy_status.report.stageType
+            self._environ['COMPUTE_ENV_TYPE'] = deploy_status.report.stageType
```

Comment:
Should we also rename the `stageType` field in the report?

Reply:
What is the difference between env `{envName}-{stageType}` and this compute env?